build.proj

```diff
@@ -138,8 +138,8 @@
   <!-- Override RestorePackages from dir.traversal.targets and do a batch restore -->
   <Target Name="RestorePackages" DependsOnTargets="BatchRestorePackages" />
 
-  <!-- Override clean from dir.traversal.targets and just remove the full BinDir -->
-  <Target Name="Clean">
+  <!-- Override CleanAllProjects from dir.traversal.targets and just remove the full BinDir -->
```

Comment:
Reviewers: this change was merged into CoreFx first and then copied here verbatim.